src/index.html

```diff
@@ -2,17 +2,43 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1.0"
-    />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Calendar</title>
-    <link
-      rel="stylesheet"
-      href="styles/index.scss"
-    />
+    <link rel="stylesheet" href="styles/main.scss" />
```

Comment:
Stylesheets with a .scss extension won't work directly in browsers. You need to compile them into .css files. Make sure you link to the correct .css file after compilation.